ocp_resources/ocs_initialization.py

```diff
@@ -0,0 +1,53 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+
+
+class OCSInitialization(NamespacedResource):
+
+    api_group = NamespacedResource.ApiGroup.OCS_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        teardown=False,
+        privileged_client=None,
+        yaml_file=None,
+        delete_timeout=TIMEOUT_4MINUTES,
+        enable_ceph_tools=None,
+        **kwargs,
+    ):
+        """
+        OCSInitialization object.
+        API reference (source code linked here due to lack of API doc for this resource):
+            https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/ocsinitialization_types.go
+
+        Args:
+            enable_ceph_tools (bool): When True, creates a Deployment named "rook-ceph-tools" in the "openshift-storage"
+             namespace. More information about the Rook Ceph Toolbox can be found here:
+             https://rook.io/docs/rook/v1.10/Troubleshooting/ceph-toolbox/
+        """
+
+        super().__init__(
+            name=name,
+            namespace=namespace,
+            client=client,
+            teardown=teardown,
+            privileged_client=privileged_client,
+            yaml_file=yaml_file,
+            delete_timeout=delete_timeout,
+            **kwargs,
+        )
+        self.enable_ceph_tools = enable_ceph_tools
+
+    def to_dict(self):
+        super().to_dict()
+        if not self.yaml_file and self.enable_ceph_tools is not None:
```

Comment:
i may have been confused before but this is what we need right?
```suggestion
        if not self.yaml_file and self.enable_ceph_tools:
```

?

Reply:
Your comment was correct, but the suggestion contained wrong code :smile: 
Your current suggestion would mean that the condition would be True only if enable_ceph_tools is true, which is what bothered you in the first place.
'not None' is the correct logic here, because I want the condition to be met when enable_ceph_tools exists, regardless of if it's True or False.

Reply:
so what confused me was the docstring:
```
enable_ceph_tools (bool): When True, creates a Deployment named "rook-ceph-tools" in the "openshift-storage"  namespace.
```
which should be updated based on your last comment

Reply:
The OCSInitialization resource always exists. I'm adding it to the wrapper so that I may apply the "enableCephTools = True" spec.
When "enableCephTools = True" is added to the spec, a new Deployment is created (aside from this OCSInitialization resource).

So to me the docstring makes sense, as well as the init function. What update would you like to see?

Reply:
If the use case is to support "enableCephTools = True" then the condition should be  `if not self.yaml_file and self.enable_ceph_tools` because there's no point in adding `"enableCephTools = False" `

Reply:
There might not be a point, but it is supported by the API.
Also, this is a boolean parameter, this both False and True should be supported, even if deleting the parameter altogether doesn't differ functionally from setting it to False.

Reply:
when setting "enableCephTools = False"; will it be present in the CR spec or will it be removed?

Reply:
if present as False then LGTM